app/views/registration/_form.erb

```diff
@@ -84,6 +84,18 @@
       <%= number_field_tag 'jwt_expiration', params[:jwt_expiration] || 4, class: 'form-control', min: 1, required: true %>
     </div>
   </div>
+  <div class='form-group'>
+    <label class='col-md-2 control-label'>
+      <%= _('Insecure?') %>
+      <% help = _('Make first curl call to the Foreman insecure. All other calls to the Foreman will be secure.') %>
```

Comment:
I'd formulate this a bit differently. `If the target machine does not trust the Foreman SSL certificate, the initial connection could be subject to Man in the middle attack. If you accept the risk and do not require the server authenticity to be verified, you can enable insecure argument for the initial curl. Note that all subsequent communication is then properly secured, because the initial request deploys the SSL certificate for the rest of the registration process.`